src/main/scala/rocket/ALU.scala

```diff
@@ -26,6 +25,46 @@ object ALU
   def FN_SLTU = UInt(14)
   def FN_SGEU = UInt(15)
 
+  // from Zb
+  // Zba: UW is encoded here becuase it is DW_64
+  def FN_ADDUW    = UInt(0)
+  def FN_SLLIUW   = UInt(0)
+  def FN_SH1ADD   = UInt(0)
+  def FN_SH1ADDUW = UInt(0)
+  def FN_SH2ADD   = UInt(0)
+  def FN_SH2ADDUW = UInt(0)
+  def FN_SH3ADD   = UInt(0)
+  def FN_SH3ADDUW = UInt(0)
+  // Zbb
+  def FN_ROR      = UInt(0)
+  def FN_ROL      = UInt(0)
+  def FN_ANDN     = UInt(0)
+  def FN_ORN      = UInt(0)
+  def FN_XNOR     = UInt(0)
+  def FN_REV8     = UInt(0)
+  def FN_ORCB     = UInt(0)
+  def FN_SEXTB    = UInt(0)
+  def FN_SEXTH    = UInt(0)
+  def FN_ZEXTH    = UInt(0)
+  def FN_MAX      = UInt(0)
+  def FN_MAXU     = UInt(0)
+  def FN_MIN      = UInt(0)
+  def FN_MINU     = UInt(0)
+  def FN_CPOP     = UInt(0)
+  def FN_CLZ      = UInt(0)
+  def FN_CTZ      = UInt(0)
+  // Zbs
+  def FN_BCLR     = UInt(0)
+  def FN_BEXT     = UInt(0)
+  def FN_BINV     = UInt(0)
+  def FN_BSET     = UInt(0)
+  // Zbk
+  def FN_BREV8    = UInt(0)
+  def FN_PACK     = UInt(0)
+  def FN_PACKH    = UInt(0)
+  def FN_ZIP      = UInt(0)
+  def FN_UNZIP    = UInt(0)
```

Comment:
why these are all 0?

Reply:
> why these are all 0?

Because ALU does not use them, and ABLU has different encoding for them